apps/server/src/modules/deletion/controller/deletion-executions.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { Controller, Post, Query, UseGuards } from '@nestjs/common';
+import { ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+import { ForbiddenOperationError, ValidationError } from '@shared/common';
+import { AuthGuard } from '@nestjs/passport';
+import { DeletionRequestUc } from '../uc/deletion-request.uc';
+import { DeletionExecutionParams } from './dto';
+
+@ApiTags('DeletionExecutions')
+@UseGuards(AuthGuard('api-key'))
+@Controller('deletionExecutions')
+export class DeletionExecutionsController {
+	constructor(private readonly deletionRequestUc: DeletionRequestUc) {}
+
+	@Post()
+	@ApiOperation({
+		summary: '"Queueing" a deletion request',
```

Comment:
I think you copied the summary from other controller by an accident, please change it to proper one.

Reply:
fixed